content/help/ops/setup/injection/index.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Sidecar Injection Webhook
+description: Describes Istio's use of Kubernetes webhooks for automatic sidecar injection.
+weight: 30
+---
+
+Automatic sidecar injection adds the sidecar proxy into user created
+pods. It uses a `MutatingWebhook` to append the sidecar’s containers
+and volumes to each pod’s template spec during creation
+time. Injection can be scoped to particular sets of namespaces using
+the webhooks `namespaceSelector` mechanism. Injection can also be
+enabled and disabled per-pod with an annotation.
+
+Whether or not a sidecar is injected is depends on three things:
+
+* webhooks `namespaceSelector`
+* default `policy`
+* per-pod override annotation
+
+The following truth table shows the final injection status based on
+these three variables.
+
+| `namespaceSelector` match | default `policy` | Pod override annotation `sidecar.istio.io/inject` | Sidecar injected? |
+|---------------------------|------------------|---------------------------------------------------|-----------|
+| yes                       | enabled          | true                                              | yes       |
+| yes                       | enabled          | false                                             | no        |
+| yes                       | enabled          |                                                   | yes       |
+| yes                       | disabled         | true                                              | yes       |
+| yes                       | disabled         | false                                             | no        |
+| yes                       | disabled         |                                                   | no        |
+| no                        | enabled          | true                                              | no        |
+| no                        | enabled          | false                                             | no        |
+| no                        | enabled          |                                                   | no        |
+| no                        | disabled         | true                                              | no        |
+| no                        | disabled         | false                                             | no        |
+| no                        | disabled         |                                                   | no        |
+
+## The result of sidecar injection was not what I expected
+
+This includes an injected sidecar when it wasn't expected and a lack
+of injected sidecar when it was.
+
+1. Check the webhook's `namespaceSelector` to determine whether the
```

Comment:
If you do want to have a bulleted list, then you need to indent all the content below by 4, until the next bullet. As it is, all your bullets are labeled with "1" in the rendered output. If you indent the content, then the bullets will be labeled with 1, 2, 3

Reply:
@geeknoid means numbered steps. :)